src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -651,6 +651,7 @@ namespace Akka.Actor
         public abstract Akka.Actor.IInternalActorRef LookupRoot { get; }
         public abstract Akka.Actor.IActorRefProvider Provider { get; }
         public abstract Akka.Actor.IInternalActorRef SystemGuardian { get; }
+        public abstract void Abort();
```

Comment:
Made `Abort` a public member, although a user will have to know to cast an `ActorSystem` to `ActorSystemImpl` to get access to this method.